graylog2-server/src/main/java/org/graylog/events/notifications/NotificationResourceHandler.java

```diff
@@ -156,6 +171,93 @@ public boolean delete(String dtoId) {
         return notificationService.delete(dtoId) > 0;
     }
 
+    /**
+     * Tests a notification definition by executing it with a dummy event.
+     *
+     * @param notificationDto the notification definition to test
+     * @param userName the name of the user that triggered the test
+     * @throws NotFoundException if the notification definition or the notification factory cannot be found
+     */
+    public void test(NotificationDto notificationDto, String userName) throws NotFoundException {
```

Comment:
The method also throws an `InternalServerErrorException` either add that to `throws` or remove both.